Tests/test_Muscle_tool.py

```diff
@@ -356,18 +390,27 @@ def test_with_multiple_output_formats(self):
         records = list(SeqIO.parse(input_file, "fasta"))
         records.sort(key=lambda rec: rec.id)  # noqa: E731
         # Prepare the command... use Clustal output (with a MUSCLE header)
-        cmdline = MuscleCommandline(muscle_exe, input=input_file,
-                                    clw=True, htmlout=output_html,
-                                    clwstrictout=output_clwstrict)
-        self.assertEqual(str(cmdline).rstrip(), _escape_filename(muscle_exe) +
-                         " -in Fasta/f002 -clw -htmlout temp_f002.html" +
-                         " -clwstrictout temp_f002.clw")
+        cmdline = MuscleCommandline(
+            muscle_exe,
+            input=input_file,
+            clw=True,
+            htmlout=output_html,
+            clwstrictout=output_clwstrict,
+        )
+        self.assertEqual(
+            str(cmdline).rstrip(),
+            _escape_filename(muscle_exe)
+            + " -in Fasta/f002 -clw -htmlout temp_f002.html"
+            + " -clwstrictout temp_f002.clw",
+        )
```

Comment:
Again, a one line string should work here.

Reply:
The coma doesn't fit :(

Reply:
Ah well, thanks for checking.